app/src/main/java/com/jerboa/ui/components/common/TimeAgo.kt

```diff
@@ -131,27 +135,126 @@ fun ScoreAndTime(
         NsfwBadge(isNsfw)
         CollapsedIndicator(visible = !isExpanded, descendants = collapsedCommentsCount)
         Spacer(modifier = Modifier.padding(end = SMALL_PADDING))
-        if (showScores) {
-            Text(
-                text = score.toString(),
-                color = scoreColor(myVote = myVote),
-                fontSize = MaterialTheme.typography.bodyMedium.fontSize.times(1.3),
-            )
+        val upvotePct = upvotePercentStr(
+            upvotes = upvotes,
+            downvotes = downvotes,
+        )
+        when (voteDisplayMode) {
```

Comment:
Main change is here. All these extra vote display modes get added to the ScoreAndTime composable, which is shown in the header bar of every comment / post.